burner-ui/src/Pages/HomePage/HistoryListEvent.tsx

```diff
@@ -63,10 +84,11 @@ const HistoryListEvent: React.FC<HistoryListEventProps> = ({ event, account, cla
   switch (event.type) {
     case 'send':
       const isReceive = event.to === account;
-      type = isReceive ? 'Receive' : 'Send';
+      type = isReceive ? '\u2199' : '\u2197';
+      // TODO: Remove line 90
       main = (
         <Fragment>
-          {isReceive ? 'From: ' : 'To: '}
+           {isReceive ? '' : ''}
```

Comment:
What's the purpose of this?

Reply:
@dmihal Thanks for catching this. That was me because I couldn't figure out a way to remove the entire line without it breaking the app.